boards/mbed_lpc1768/system.c

```diff
@@ -1,478 +0,0 @@
-/**************************************************************************//**
```

Comment:
why is this file removed?

Reply:
oops, just for fun? 😄

Reply:
sounds good :)

Reply:
ah no, because its basically the same file as for the seeedoino and that I moved to the cpu/lpc1768 so its shared with the mbed board, hence the remove.

Reply:
ok, cool! :)